### PR TITLE
Introduce a dynamic config for cassandra all consistency level delete

### PR DESCRIPTION
**Pull request by Shaddoll**
**What changed?**
Introduce a dynamic config for cassandra all consistency level delete

**Why?**
feature flag

**How did you test it?**

**Potential risks**

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: common/persistence/nosql/nosqlConfigStore.go [hunk 1]
```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/uber/cadence/common/config"
 	"github.com/uber/cadence/common/log"
 	"github.com/uber/cadence/common/persistence"
+	p "github.com/uber/cadence/common/persistence"
```
Comment (ZackLK) [inline common/persistence/nosql/nosqlConfigStore.go:30]: Why did you add the same import a second time?

**Review by ZackLK — APPROVED** [review]

File: common/persistence/nosql/nosqlConfigStore.go (hunk 1 above)
Comment (Shaddoll) [reply common/persistence/nosql/nosqlConfigStore.go:30]: I didn't see the previous one.
